### DISCUSSION TITLE
Disable touch screen for when it's raining

### DISCUSSION THREAD

**Discussion by zo0om** [organicmaps/organicmaps#6135] [Ideas]
There I was navigating along quite happily, using my "waterproof" phone on my handlebars, then it rained and things started to go haywire. I read that you can download apps to disable the touch screen but it would be great if this were a built-in feature!

**Comment by biodranik** [comment]
1. How the app should be used without a working touch screen?
2. How to detect that the phone screen is covered by water?

**Comment by zo0om** [comment]
1. You won't need any controls if you're just following directions. The touch screen can be re-enabled by pressing a physical button.
2. There is no need to do that - it will be a manual operation to disable the touch screen.

**Reply by biodranik** [reply]
Do you know any apps that have this feature? What should be the key/combination to disable and enable touches in a convenient (and ideally obvious) way?

**Reply by zo0om** [reply]
[This](https://www.makeuseof.com/tag/disable-touchscreen-input-android-iphone/) is the page I found when I looked into it after I had the problem. It seems the apps require permission to be displayed on top of other apps, then are transparent but steal all the touches. Of course that permission wouldn't be required if the feature is built-in.

I would think that a menu option could disable the touch screen, and perhaps pressing any physical button would re-enable it again. I don't know if you're allowed to override the power button, but it wouldn't really matter if you weren't (the screen would just go off) so long as you could detect it being pressed.
